File: tag2/b3k.py
#!/usr/bin/env python 
from sys import stdin, stderr


for line in stdin:
    line = line.strip()
    if "error" in line:
        stderr.write("stderr: " + line + "\n")
    elif "stdout" in line:
        print "stdout: " + line
    else:
        print "Echo: " + line

File: tag2/b3k.pl
#!/usr/bin/env perl


while($line = <>) {
    chomp $line;
    if (index($line, 'error') != -1) {
        print STDERR "stderr: ".$line."\n";
    } elsif (index($line, 'stdout') != -1){
        print "stdout: ".$line."\n"; 
    } else {
        print "Echo: ".$line."\n";  
    }
}
